j/jaf-api/jaf-api_2.1.1_ubi_8.5.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: jaf-api
+# Version	: 2.1.1
+# Source repo	: https://github.com/jakartaee/jaf-api
+# Tested on	: UBI 8.5
+# Language      : Java, Html, Ruby
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Muskaan Sheik <Muskaan.Sheik@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=jaf-api
+PACKAGE_VERSION=${1:-2.1.1}
+PACKAGE_URL=https://github.com/jakartaee/jaf-api
+
+yum -y update
+yum install -y gcc-c++ gcc npm wget curl unzip nano make git
+
+MAVEN_VERSION=3.6.3
+wget http://mirrors.estointernet.in/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+tar -C /usr/local/ -xzf apache-maven-$MAVEN_VERSION-bin.tar.gz
+mv /usr/local/apache-maven-$MAVEN_VERSION /usr/local/maven
+rm apache-maven-$MAVEN_VERSION-bin.tar.gz
+export M2_HOME=/usr/local/maven
+export PATH=$PATH:$M2_HOME/bin
+
+yum install -y java-11-openjdk java-11-openjdk-devel java-11-openjdk-headless
+ export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-11.0.18.0.10-2.el8_7.ppc64le
```

Comment:
@muskaan2199 We can set like `export JAVA_HOME=/usr/lib/jvm` and not specific version.
Please check

Reply:
@nikhil-kalbande I tried. It gives me an error